README.md

```diff
@@ -53,9 +53,14 @@ This documentation provides a comprehensive overview of the UKEF Digital TradeFi
 6. Set UKEF TFM environment variables in your terminal: `UKEF_TFM_API_SYSTEM_KEY` and `UKEF_TFM_API_REPORTS_KEY`.
 7. Run `npm run env:copy` to copy your root .env file into all the individual projects that need it.
 8. Run `npm ci` in the root folder of the repository. (note: this will install dependencies for the entire project, including those specified in sub-packages. More details on this in the [npm workspaces](./doc/npm-workspaces.md) docs)
-9. Start your local environment with `npm run start`.
-10. Run migrations on the MSSQL Server database (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
-11. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md))
+9. Start your local environment with `docker-compose up --build`.
+10. Create mock data
+    - If not working on utilisation reports:
+      1. Run migrations on the MSSQL Server database by running `npm run db:migrate` in the `libs/common` directory (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
+      2. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md)).
+    - If working on utilisation reports or E2E tests for utilisation reports:
+      1. Run the command `npm run load:init` from the root folder of the repository.
+      - This runs `npm run db:migrate` in the `libs/common` directory which creates the SQL database tables. This is followed by `npm run load:sql` in the `utils` directory which inserts mock data for MongoDB and MSSQL. This is especially useful for utilisation report work and E2E tests
```

Comment:
Do we need this step? If it's explained in the `run-migrations` README I don't think we need it here, adds confusion IMO

Reply:
I added this to explain what the command does so there is no confusion.  Not sure how to reword it

Reply:
I think this is fairly complicated for a mock data loader, I would expect `npm run load:sql` to be executed as part of `npm run load` otherwise for a new developer it is fairly time consuming.

Appreciate not part of this PR but whilst we at it we are ought to simplify this please.